client/presenterControl/public/components/asqw-connection/_asqw-connection.js

```diff
@@ -17,6 +17,7 @@ Polymer({
   ],
 
   events2Forward: [
+    "asq:sessionEventBeamer",
```

Comment:
What you want to do is go and add this line here: https://github.com/ASQ-USI/ASQ/blob/devel/client/js/presenter.js#L89